### PR TITLE
"confirm tx on main net" restyled

### PR DESCRIPTION
**Pull request by LianaHus**
fix https://github.com/ethereum/remix-project/issues/1048

### CODE REVIEW

**Review by ryestew — COMMENTED** [review]
Right now in this PR the modal looks like this:
image

Here's a shorter line:
image

File: apps/remix-ide/src/app/ui/confirmDialog.js [hunk 1]
```diff
@@ -25,24 +25,48 @@ function confirmDialog (tx, amount, gasEstimation, self, newGasPriceCb, initialP
...
-      <div>To: ${tx.to ? tx.to : '(Contract Creation)'}</div>
-      <div>Amount: ${amount} Ether</div>
-      <div>Gas estimation: ${gasEstimation}</div>
-      <div>Gas limit: ${tx.gas}</div>
-      <div>Gas price: <input id='gasprice' oninput=${onGasPriceChange} /> Gwei <span> (visit <a target='_blank' href='https://ethgasstation.info'>ethgasstation.info</a> to get more info about gas price)</span></div>
-      <div>Max transaction fee:<span id='txfee'></span></div>
-      <div>Data:</div>
-      <pre class=${css.wrapword}>${tx.data && tx.data.length > 50 ? tx.data.substring(0, 49) + '...' : tx.data} ${copyToClipboard(() => { return tx.data })}</pre>
+    <div>
+      <div>You are about to create a transaction on the main network. Confirm details to send to your provider.</div>
```
Comment (ryestew) [inline apps/remix-ide/src/app/ui/confirmDialog.js:29]: <div>You are about to create a transaction on the main network. <br>Confirm the details to send to your provider.</div>

File: apps/remix-ide/src/app/ui/confirmDialog.js [hunk 2]
```diff
@@ -25,24 +25,48 @@ function confirmDialog (tx, amount, gasEstimation, self, newGasPriceCb, initialP
...
+          <span>${tx.gas}</span>
+        </div>
+        <div>
+          <span class="text-dark mr-2">Max transaction fee:</span>
+          <span id='txfee'></span>
+        </div>
+        <div class="d-flex align-items-center my-1">
+          <span class="text-dark mr-2">Gas price:</span>
+          <input class="form-control mr-1" style='width: 40px; height: 28px;'id='gasprice' oninput=${onGasPriceChange} />
+          <span>Gwei (visit <a target='_blank' href='https://ethgasstation.info'>ethgasstation.info</a> to get more info about gas price)</span>
```
Comment (ryestew) [inline apps/remix-ide/src/app/ui/confirmDialog.js:58]: <span>Gwei (visit <a target='_blank' href='https://ethgasstation.info'>ethgasstation.info</a> for current gas price info)</span>

**Review by ryestew — CHANGES_REQUESTED** [review]

**Comment by LianaHus** [comment]
@ryestew done!
thanks

**Comment by LianaHus** [comment]
@ryestew could you please review again? TIA

**Comment by yann300** [comment]
changes sounds good, waiting for @ryestew approval

File: apps/remix-ide/src/app/ui/confirmDialog.js [hunk 3]
```diff
@@ -25,24 +25,48 @@ function confirmDialog (tx, amount, gasEstimation, self, newGasPriceCb, initialP
...
-      <div>To: ${tx.to ? tx.to : '(Contract Creation)'}</div>
-      <div>Amount: ${amount} Ether</div>
-      <div>Gas estimation: ${gasEstimation}</div>
-      <div>Gas limit: ${tx.gas}</div>
-      <div>Gas price: <input id='gasprice' oninput=${onGasPriceChange} /> Gwei <span> (visit <a target='_blank' href='https://ethgasstation.info'>ethgasstation.info</a> to get more info about gas price)</span></div>
-      <div>Max transaction fee:<span id='txfee'></span></div>
-      <div>Data:</div>
-      <pre class=${css.wrapword}>${tx.data && tx.data.length > 50 ? tx.data.substring(0, 49) + '...' : tx.data} ${copyToClipboard(() => { return tx.data })}</pre>
+    <div>
+      <div>You are about to create a transaction on the main network. Confirm the details to send to your provider.</div>
```
Comment (ryestew) [inline apps/remix-ide/src/app/ui/confirmDialog.js:29]: For many users, I think the word **provider** is not clear.  Is this line better: 

You are about to create a transaction on the main network. Confirm the details to send to your provider.  The provider for many users is MetaMask.  Confirming the details only sends the info to the provider -where in turn, you will need to sign the transaction to send it to the main network.

or 

You are about to create a transaction on the main network. Confirm the details to send the info to your provider.  The provider for many users is MetaMask.  The provider will ask you to sign the transaction before it is sent to the main network.

**Review by ryestew — APPROVED** [review]

**Pull request merged**